docs/about.md

```diff
@@ -14,13 +14,20 @@ The **project team**:
 * [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
 * [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
 * [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
-* [**Damith C. Rajapakse**](https://github.com/damithch): _Project Mentor_ since Aug 2016
-* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_  Jan 2018 - Aug 2018
+* [**Damith C. Rajapakse**](https://www.comp.nus.edu.sg/~damithch/): _Project Mentor_ since Aug 2016
 * [**Jamos Tay**](https://github.com/jamos-tay): _Project Member_ since Aug 2018
-* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ May 2018 - Aug 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
-* [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ since Jan 2018
+* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ since Jan 2019
+* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ since Jan 2019
+* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ since Jan 2019
+* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Member_ since Jan 2019
 * [**Tan Wang Leng**](https://github.com/yamgent): Current _Team Lead_ since Jul 2018, _Project Member_ for May 2018 - Jun 2018
+* [**Tan Zhen Yong**](https://www.github.com/Xenonym): _Project Member_ since Jan 2019
+
+Past Members:
+* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_ Jan 2018 - Aug 2018
```

Comment:
Please help to standardize the `for` keyword before time ranges. Not sure if having or not is better. :see_no_evil:

Reply:
Will do 👍